crates/moonbuild/src/entry.rs

```diff
@@ -551,16 +551,16 @@ pub fn run_test(
 
             let printed = Arc::clone(&printed);
             handlers.push(async move {
-                let mut result = trace::scope("test", || async {
+                let mut result = trace::async_scope(
+                    "test",
```

Comment:
we could sepecify which test is running

Reply:
Several names which are passed to `trace::scope` need to be adjusted/polished. It is good to fix them in another patch.